misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickLineScreen.jsx

```diff
@@ -1,8 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
 import PickStepButton from './PickStepButton';
 
+const normalizePickFromAlternatives = (pickFromAlternatives) => {
+  let isArrayPFA = Array.isArray(pickFromAlternatives);
+  let normalizedAlts = {};
+  if (isArrayPFA) {
```

Comment:
which is the case when it's not an array?
why is this not normalized when we merge it to state? why here, every time?

Reply:
wanted to do more investigation on this but I do not have the time. This is a sol I did under time pressure to silent that warning got in console. I know it is not ok. 

So how to proceed ? Leave the warning alone and do more investigation when we have time (removing this code also) ?

Reply:
found it, it was a chaotic mix of data.